src/NodeVisitor/EscaperNodeVisitor.php

```diff
@@ -157,7 +157,7 @@ private function isSafeFor(string $type, AbstractExpression $expression, Environ
         return \in_array($type, $safe) || \in_array('all', $safe);
     }
 
-    private function needEscaping()
+    private function needEscaping(): string|bool
```

Comment:
`string|false` actually. `true` is not a valid strategy.

Reply:
"true" will wait Twig4 / PHP 8.2 no ?

Reply:
I don't understand what you mean here. `true` would indeed require PHP 8.2 to be used. But the one we need here is `false`, not `true`